src/assets/stylesheets/index.scss

```diff
@@ -2107,67 +2075,6 @@ img.cookie-img {
   }
 }
 
-/* Text image widget ************ TBD ************
-=================================================================== */
-.text_image_widget {
-  overflow: hidden;
-}
-
-/* floating-image ************ TBD ************
-=================================================================== */
-
-.floating-image.pull-left {
-  margin: 0 20px 5px 0;
-}
-.floating-image.pull-right {
-  margin: 0 0 5px 20px;
-}
-
-.floating-image img {
-  margin: 0;
-}
-
-.width-25 {
-  width: 25%;
-}
-.width-33 {
-  width: 33%;
-}
-.width-50 {
-  width: 50%;
-}
-.width-66 {
-  width: 66%;
-}
-.width-75 {
-  width: 75%;
-}
-.width-100 {
-  width: 100%;
-}
-
-.floating-image.width-25,
-.floating-image.width-33,
-.floating-image.width-50,
-.floating-image.width-66,
-.floating-image.width-75,
-.floating-image.width-100 {
-  img {
-    width: 100%;
-  }
-}
-
-/* google maps fix
-================================================== */
-
-.gm-style img {
```

Comment:
Actually `gm-style` is used indirectly in the `GoogleMapsWidget`, if you set it to `Interactive` (Widget property).

Can you move this section to `GoogleMapsWidget.scss`?

Reply:
oh oki